controller_manager/test/test_hardware_resources/test_actuator_hardware.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 ros2_control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+
+#include "./test_actuator_hardware.hpp"
+
+hardware_interface::return_type
```

Comment:
since you are in a cpp, I'd put a `namespace hardware_interface` around the contents of this file and save a bunch of namespacing

Reply:
I disagree. It's a plugin which ideally lives in a separate package. I can add some `using` directives to make it shorter if you wish.

Reply:
Good point. Let's drop some `using`s or a single `using namespace hardware_interface`, the scope is small enough for that to be safe.